doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -30,10 +56,51 @@
 > * Which optional features are used and which optional API-s are available
 > * Which [alfa or beta features](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) are used
 
+The distribution and version of Kubernetes used must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) which lists all those that are Certified as being conformant with the CNCF Kubernetes specification.
```

Comment:
- This document starts to have versions from 1.6. I believe we should have a more recent Kubernetes version required.
  - With this statement we require the CNCF Certification what excludes those who are not CNCF members and do not want to pay a lot for the certification.

Reply:
> * This document starts to have versions from 1.6. I believe we should have a more recent Kubernetes version required.

Happy to specify a more recent version (and am expecting to) but it needs to be driven by the requirements - i.e. if there is a requirement that can only be met by a newer set of versions.  I plan to do that based on the feature gates etc. that I've already specified as they won't all be in those older versions.

Reply:
> * With this statement we require the CNCF Certification what excludes those who are not CNCF members and do not want to pay a lot for the certification.

Yes - they must either be a CNCF member or willing to pay a fee.  I think it's important to state this conformance even so.

Reply:
the basic requirement is that nobody should ever use a non-supported version, as there will be no security fixes for it.
so in K8s case it is a must have for an infra to use at least N-2 major version, and the latest minor version

BTW features required by us are coming out continuously, for example usable NUMA allocation in 1.17, NUMA aware hugepages possibly in 1.18 / 1.19 etc.
New is always better

Reply:
Good point.  Have added the following to row60:

>In order to align with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), a Reference Implementation must use one of three latest minor versions (`n-2`) - e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15.

Does that resolve the change request @CsatariGergely?